client/js/socket-events/search.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
+const $ = require("jquery");
+const socket = require("../socket");
+const render = require("../render");
+const messages = $("#search-query .results .messages");
+const form = $("#search-query form");
+const select = $("#search-query form select");
+
+socket.on("search-networks-channels", function(data) {
+	var groups = {};
```

Comment:
let

Reply:
Eslint advise to use const in that case, what's your opinion?